pkg/guestagent/guestagent_linux.go

```diff
@@ -326,6 +326,10 @@ func (a *agent) fixSystemTimeSkew() {
 	for {
 		ticker := time.NewTicker(10 * time.Second)
 		for now := range ticker.C {
+			if !timesync.DeviceExists() {
+				logrus.Infof("fixSystemTimeSkew: device does not exist")
+				break
```

Comment:
Apparently the "break" is **not** enough to exit, it is like the range has queued a lot of events already?